specifyweb/frontend/js_src/lib/components/Forms/Save.tsx

```diff
@@ -192,6 +207,8 @@ export function SaveButton<SCHEMA extends AnySchema = AnySchema>({
   // Don't allow cloning the resource if it changed
   const isChanged = saveRequired || externalSaveRequired;
 
+  const cannotClone = resourcesToNotClone.has(resource.get('name') || '');
```

Comment:
do not pollute the Save button with random logic from different places
at very least, move this logic into the `useEnabledButtons` hook as it contains the logic for similar things

Reply:
> do not pollute the Save button with random logic from different places

to clarify on this, the Save button component should be as generic as possible, able to handle all record types
if each time there is a special case in some table we add code to Save (or ResourceView), the code would become unmanageable (not to mention bug prone - what if the `name` field is present in some other table that matches your condition? - you should explicitly check for table name first)
so ideally there would be no table-specific logic, or it would be defined up in some sort of AppResource component in a way that would apply in the save button here. but if that is not easy then at very least the component-specific logic should be clearly isolated away - that's what the useEnabledButtons hook is for